src/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
     "strict": true,
-    "module": "CommonJS",
-    "target": "es2018",
+    "module": "ESNext",
```

Comment:
This will cause tsc to output ESM code, is that correct? IIRC `vercel build` does not yet support ESM modules for Builders, so I'm not sure this would work.

Reply:
That's why I moved to ncc instead of tsc to build the bundled CJS file.

Reply:
I'd prefer to stick with `tsc`. Can we revert this tsconfig change?